operator/README.md

```diff
@@ -0,0 +1,89 @@
+# Kubernetes Operator for Nessie
+
+## Overview
+
+This module is a Kubernetes Operator for Nessie.
+
+**WARNING: This is a work in progress and is not ready for production use.**
+
+## Usage
+
+TODO describe typical deployment scenarios.
+
+## Development
+
+### Prerequisites
+
+Install the following tools:
+
+- Kubectl: https://kubernetes.io/docs/tasks/tools/
+- Operator SDK: https://sdk.operatorframework.io/docs/installation/
+- Optional: 
+  - yq: https://github.com/mikefarah/yq
+
+**IMPORTANT**: The current Operator SDK version, 1.33.0, has [a bug] that prevents it from working
+with the latest version of the Kubernetes API (or Quarkus plugin). To work around this, install the
+version 1.32.0 of the operator-sdk.
+
+[a bug]:https://github.com/operator-framework/operator-sdk/issues/6645
+
+Using brew:
+
+```bash
+wget https://raw.githubusercontent.com/Homebrew/homebrew-core/6b039d0d8907b70704cd1d87ad235705a77620ae/Formula/o/operator-sdk.rb
```

Comment:
Yes, that's the URL for 1.32.0. But you don't need to worry, `operator-sdk` CLI is only needed when creating the project, or when adding / modifying CRDs. You don't even need to install it since the project is created already, and there won't be more CRDs any time soon.

Reply:
Isn't there some `brew install operator-sdk` or so?

Reply:
My concern's more that this info gets outdated very quickly and finding the "right" link in a couple of months will be tricky.

Reply:
Yes, of course you can run `brew install operator-sdk`. You will get operator-sdk 1.33 – with which I had a few issues when bootstrapping the project. But now that the project is bootstrapped, we don't need it anymore; so I will remove this section of the README.